src/ApolloHooksQuery.re

```diff
@@ -88,7 +88,7 @@ let useQuery:
       ~errorPolicy: ApolloHooksTypes.errorPolicy=?,
       ~skip: bool=?,
       ~pollInterval: int=?,
-      (module Config with type t = t)
+      graphqlDefinition(t, _, _)
```

Comment:
We can have inline types now that we don't use first class modules anymore, making the code less verbose.

Reply:
I think @fakenickels wanted to create interfaces for each file, maybe it will be easier to break out the types if they aren't inline in that case?